lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -451,6 +451,9 @@ def _configure_node(
             self.vm_disks_dir, f"{node_context.vm_name}-os.qcow2"
         )
 
+        # if node_runbook.disk_size_gb:
```

Comment:
What's with this commented out code?

Reply:
Removed. This code i kept as it is to get the resize param from runbook.